### DISCUSSION TITLE
Alerts not properly acknowledged

### DISCUSSION THREAD

**Discussion by launchdaemon** [Security-Onion-Solutions/securityonion#5201] [Unsupported Versions] [answered]
Hi,

Firstly I would like to say thank you to the team for the incredible work and amazing tools you put out, Security Onion is awesome!

I have a standalone instance of Security Onion 2.3.61 installed on Ubuntu and have come across some odd behaviour which I'm not sure if it is down to me having an unintended workflow or a small bug.

From the main dashboard, where alerts are grouped by name, module and severity, I click on an alert to drill down and see the specific instances, e.g. what hostnames were queried for from a suspicious tld.  I then click on the severity and choose to group by.  The problem is, if I acknowledge them from there, they will disappear however if I refresh the page or return to the main dashboard, they will be there again and not acknowledged.  I have tried this with a very small number of alerts, most recently only two, so it should not be that it is still processing them.

If I drill down in the specific instances of an alert I can acknowledge them individually or from the main dashboard just grouped by name, module and severity again I can acknowledge them no problem.

**Comment by jertel** [comment, marked as answer]
Thanks for reporting this. #5221 was created to investigate this.

As a work-around, if you go back to the initial alerts query (group by name, module), and then, instead of using the Drilldown quick action, click the _Include_ quick action. This will give you the equivalent view that you landed on but will still allow the Ack to function.

**Reply by launchdaemon** [reply]
Hi @jertel 
Thank you for the reply and for the tip that's great and will keep an eye on that issue.
